.circleci/config.yml

```diff
@@ -55,12 +55,12 @@ jobs:
             # is now 16 and some of JMUI's dependencies break with Node 14+, so install Node 12 explicitly.
             #
             # npm WARN deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
```

Comment:
Nit: Is this comment completely outdated now? I don't see any other instances of `chokidar` in the repo.

Reply:
It's in `package-lock.json`. I went back and forth about whether to remove this... it's a legitimate problem, but only for development (chokidar is a dev-only dependency), and it's one of many scary warnings output by npm. I think it's probably better to remove it, though, and will do so.

Reply:
Oh, if it actually _is_ still used and hasn't been updated, then we should probably update it? Or change the comment to say that `chokidar` is now broken until it gets updated.

Reply:
I'll add it to `Dockerfile.dev`, since it's more relevant there.